.github/workflows/PublishWidgetData.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish widget data
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - packages/pluggableWidgets/**/*
+      - packages/customWidgets/**/*
```

Comment:
If I understand the script correctly, we only care about the `xml` files. So we could make this even more specific so it only runs when it should. Something like;
- `packages/pluggableWidgets/**/src/*.xml`
- `packages/customWidgets/**/src/*.xml`

Reply:
Good one. They don't document filter syntax to do e.g. `*.{xml,json}`, so I've listed the package.json and *.xml files separately